var/spack/repos/builtin/packages/py-flit-core/package.py

```diff
@@ -23,10 +24,13 @@ class PyFlitCore(PythonPackage):
     version("3.0.0", sha256="a465052057e2d6d957e6850e9915245adedfc4fd0dd5737d0791bf3132417c2d")
     version("2.3.0", sha256="a50bcd8bf5785e3a7d95434244f30ba693e794c5204ac1ee908fc07c4acdbf80")
 
-    # Dependencies listed in flit_core/build_thyself.py
+    # pyproject.toml
+    depends_on("python@3.7:", when="@3.7:", type=("build", "run"))
```

Comment:
`flit_core` should still work on Python 3.6 - we have CI that aims to test this. If you've found that it doesn't, please let me know.

I try to be fairly conservative with `flit_core`, while sometimes `flit` (the CLI) requires a newer version of Python.

Reply:
Good catch, not sure where I got that from